app/instance-initializers/i18n.js

```diff
@@ -0,0 +1,25 @@
+import Ember from "ember";
+
+export default {
+  name: 'i18n',
+
+  initialize: application => {
+    let i18n = application.lookup('service:i18n');
+    const locale = determineLocale(i18n.get('locales'));
+    i18n.set('locale', locale);
+    moment.locale(locale);
+    i18n.on('missing', (locale, key, context) => {
+      Ember.Logger.warn("Missing translation: " + key);
+    });
+  }
+};
+
+function determineLocale(availableLocales) {
+  let userLanguages = navigator.languages || [];
+  userLanguages.push(navigator.language, navigator.userLanguage, 'en');
+
+  return userLanguages.compact().find( language => {
+    return availableLocales.indexOf(language) > -1 ||
```

Comment:
I think we can use [`Array.prototype.includes()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes?v=control) here.